src/main/java/net/snowflake/client/core/QueryContextDTO.java

```diff
@@ -0,0 +1,36 @@
+package net.snowflake.client.core;
+import java.util.List;
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class QueryContextDTO {
```

Comment:
/**
 * The POJO object used by both JDBC and the Cloud service to exchange opaque informations.
 */